code/datums/elements/kneejerk.dm

```diff
@@ -0,0 +1,76 @@
+/// An element which enables certain items to tap people on their knees to measure brain health
+/datum/element/kneejerk
+	element_flags = ELEMENT_BESPOKE | ELEMENT_DETACH
+	id_arg_index = 2
+
+/datum/element/kneejerk/Attach(datum/target)
+	. = ..()
+
+	if (!isitem(target))
+		return ELEMENT_INCOMPATIBLE
+
+	RegisterSignal(target, COMSIG_ITEM_ATTACK, .proc/on_item_attack)
+
+/datum/element/kneejerk/Detach(datum/source, ...)
+	. = ..()
+
+	UnregisterSignal(source, COMSIG_ITEM_ATTACK)
+
+/datum/element/kneejerk/proc/on_item_attack(datum/source, mob/living/target, mob/living/user, params)
+	SIGNAL_HANDLER
+
+	var/list/modifiers = params2list(params)
+
+	///target a leg and right click with a capable item while the target is buckled
+	if((user.zone_selected in list(BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)) && LAZYACCESS(modifiers, RIGHT_CLICK) && target.buckled)
```

Comment:
Do direct `==` checks, creating a list for this is needlessly expensive--I see this in a few other places and it's wrong.

Also, your comment is not needed--it's just writing out the code directly.

Reply:
Ideally alt attacks would be their own signal as well, but that's something I'll need to fix later (none of the secondary stuff calls `..()`, so not all signals are picked up).